en/minecraft_default_config.md

```diff
@@ -4,239 +4,234 @@ title: Server Config
+With this option you can set the default game mode on your server. The game mode is given to each player when they first visit the server and can be changed at any time thereafter using the (/gamemode) [https://minecraft.gamepedia.com/Gameplay#Game_modes] command.
 
-Sofern du die IDs der jeweiligen Spielmodi nicht kannst, findest du hier eine Auflistung:
+If you do not know the IDs of the respective game modes, you will find a list here:
 
-| Wert | Spielmodus |
+| Value | Game Mode |
 |--|--|
-| 0 | Überleben |
-| 1 | Kreativ |
-| 2 | Abenteuer |
-| 3 | Zuschauer |
+| 0 | Survival |
+| 1 | Creative |
+| 2 | Adventure |
+| 3 | Spectator |
 
 ### server-port
 
-Du kannst hier  deinem Server einen festen Port zuweisen, über welchen dieser in Verbindung mit der IP erreichbar sein wird.
-Sofern du als Beispiel mehrere Server auf deinem Rootserver betreiben möchtest, muss je Server ein unterschiedlicher Port festgelegt werden. Diese Option kann in der Regel unberührt bleiben und wird meist nur bei einem BungeeCord-Netzwerk benötigt.
+Here you can assign a fixed port to your server, over which it will be accessible in connection with the IP. If you want to run several servers on your root server, you have to define a different port for each server. This option can usually be left untouched and is usually only needed for a BungeeCord network.
 
 ### enable-command-block
 
-Damit können (Befehlsblöcke)[https://minecraft-de.gamepedia.com/Befehlsblock] auf dem Server aktiviert werden.
-Sofern keine gebraucht werden, sollte der Wert auf *false* gesetzt werden, da es einige Methoden gibt, womit der Server zerstört oder beschädigt werden kann in Verbindung mit Befehlsblöcken.
+This allows (command blocks) [https://minecraft.gamepedia.com/Command_Block] to be activated on the server.
+If none are needed, the value should be set to *false*, as there are some methods that can destroy or damage the server in connection with command blocks.
 
 ### allow-nether
 
-Diese Option de- oder aktiviert werden Nether.
-Sofern deaktiviert, kann der Nether nicht mehr über ein Portal betreten werden.
-Der Zugang ist jedoch weiterhin über ein Plugin wie (Multiverse Core)[https://www.spigotmc.org/resources/multiverse-core.390/].
+This option can be enabled or disabled Nether.
+If deactivated, the Nether can no longer be entered via a portal.
+However, access is still possible via a plugin like (Multiverse Core) [https://www.spigotmc.org/resources/multiverse-core.390/].
 
 ### enable-rcon
 
-Über RCON können von überall Befehle in der Konsole ausgeführt werden.
-Nötig dazu ist ein Passwort, soweit ein separater Port. Die Einstellungen dazu werden ebenfalls hier in der Config gesetzt.
-Wir empfehlen RCON zu deaktivieren, um Eingriffe von unbefugten vorzubeugen.
+Via RCON, commands can be executed from anywhere in the console.
```

Comment:
"from anywhere in the console.", not "can be executed in the console, from anywhere"?